### ISSUE TITLE
Color contrast in the interfaces tab is to be changed, text color should be black.

### ISSUE THREAD

**Issue by andreagiaquinto** [jerrykuku/luci-theme-argon#554]
![Image](https://github.com/user-attachments/assets/a250f2c4-fef0-4ebb-b08e-33f03187baf0)

In te intercaces tab, ss you can see from the image, the text over color background has bad contrast, text should be black.

Regards
Andrea

**Comment by jerrykuku** [comment]
Known colors have been darkened in dark mode, but I cannot reproduce the colors above you, and I have to understand the specific code to make the adaptation.

**Comment by andreagiaquinto** [comment]
Hello @jerrykuku, you can add several firewall zone and setup every interface for each zone.
Just for info, in the light mode with the same colors but the label text is black, so is correct.

Regards
Andrea

**Comment by sbwml** [comment]
白色大法好

![image](https://github.com/user-attachments/assets/3b112bbe-d842-41a0-8e28-c7776463bdb8)

**Comment by constgen** [comment]
To reproduce it you need to use non standart names of interfaces. These is the same config on 2 themes

| Argon | Bootstrap |
|--------|--------|
| <img width="120" alt="Image" src="https://github.com/user-attachments/assets/c088a260-7485-467f-aff7-fc0c1a96794c" /> | <img width="203" alt="Image" src="https://github.com/user-attachments/assets/dbade558-6401-40ec-921e-dcb09406fa6d" /> |

I think about 2 options as a solution
- text shadow and dark semiopacity overlay on the background (e.g. using pseudo element)
- CSS Mix Blend Mode https://css-tricks.com/methods-contrasting-text-backgrounds/

**Comment by constgen** [comment]
Did some experiments. Think I found the necessary styles for Interfaces labels. These are examples with new styles applied to `lan` and `wan`. It dinamically adjust text color to background color to be more contrast. Pay attention that even `wan` is more vosoble than `wan6` or `WISP`

| Fixed | Current |
|--------|--------|
| ![Image](https://github.com/user-attachments/assets/a8121a0b-92aa-468d-a0cc-74ecba5085ef) | ![Image](https://github.com/user-attachments/assets/bca1d676-6c33-46a1-b68e-f4f7476f9c8e) |

These are the styles 
```css
filter: invert(1) grayscale(1) contrast(9);
color: transparent;
background: inherit;
background-clip: text;
```
